step-ca/certificate-authority-server-production.mdx

```diff
@@ -175,112 +174,173 @@ So do some certificates.
 But it can be difficult to operationalize such short-lived certificates.
 </Footnote>
 
-### Enable Active Revocation On Your Intermediate CA
+### Consider Active Revocation
 
-The value of a two-tiered PKI is that you can add your root CA certificate to the certificate trust store on all of your nodes, and store your root private key completely offline.
-A leaf certificate signed by the CA always comes in a bundle that contains the intermediate CA certificate alongside the leaf certificate.
-With this bundle, any client that trusts your root CA can verify the complete chain of trust.
+You may require Active Revocation if you need immediate certificate revocation,
+or if you are issuing long-lived certificates.
+For this purpose,
+`step-ca` contains a built-in,
+minimalist Certificate Revocation List (CRL) server.
+
+This section describes how to enable CRL for your intermediate CA and leaf certificates.
+
+<Alert severity="info">
+  <div>
+    The support for active revocation in `step-ca` is designed to be minimalistic.
+    If you need OCSP, check out Smallstep's <a href="https://smallstep.com/certificate-manager/">commercial offerings</a>.
+  </div>
+</Alert>
+
+#### When To Use Active Revocation?
+
+The value of a two-tiered PKI is in the decoupling of root and intermediate CAs.
+You can add your root CA certificate to the certificate trust store on all of your nodes,
+and store your root private key completely offline.
+When `step-ca` issues a certificate to a client,
+it comes inside a PEM bundle that contains both the intermediate CA certificate(s) and the end entity certificate.
```

Comment:
There's a few lines with lowercase intermediate, and a few where it's capitalized

Reply:
Yes, sometimes with longer sentences I put one sentence fragment per line. It makes diffs easier to read.
The rendered file puts everything together into a single paragraph. So, the user doesn't see this.

Reply:
My remark was about `Intermediate CA` vs. `intermediate CA`, and it thus not being consistent

Reply:
Oh sorry, I missed that. Good catch!